### PR TITLE
Adding Sonar wrapper 

### PR DESCRIPTION
**Pull request by babineaum**
This wrapper checks the job specific checkbox that allows for sonar env vars to be populated and used during the build time.

### CODE REVIEW

**Comment by babineaum** [comment]
tested here https://jenkins-test.devaws.dataxu.net/job/jenkins-jobs_update_single_job/1452/ with this branch and this PR from jenkins-jobs - https://github.com/dataxu/jenkins-jobs/pull/1448

File: setup.cfg [hunk 1]
```diff
@@ -257,6 +257,7 @@ jenkins_jobs.wrappers =
     release=jenkins_jobs.modules.wrappers:release
     rvm-env=jenkins_jobs.modules.wrappers:rvm_env
     sauce-ondemand=jenkins_jobs.modules.wrappers:sauce_ondemand
+    sonar-prepare=jenkins_jobs.modules.wrappers:sonar_prepare
```
Comment (careyrp) [inline setup.cfg]: Why "sonar-prepare" instead of just "sonar"?

Comment (babineaum) [inline setup.cfg]: i dont know :) ill make it whatever
just figured sonar might make it a bit confusing, like there was some configuration available, but it just a check box

File: jenkins_jobs/modules/wrappers.py [hunk 2]
```diff
@@ -1521,6 +1521,21 @@ def generic_artifactory(parser, xml_parent, data):
         str(data.get('discard-build-artifacts', True)).lower()
 
 
+def sonar_prepare(parser, xml_parent, data):
+    """yaml: Sonar Prepare
+    Collect sonar information from configuration tab and make available via envinonment variables.
```
Comment (careyrp) [inline jenkins_jobs/modules/wrappers.py]: pep8: Line too long

**Comment by careyrp** [comment]
:shipit:

**Comment by ferrants** [comment]
looks good to me 👍

**Pull request merged**